docs/ibis-for-pandas-users.ipynb

```diff
@@ -31,6 +31,7 @@
    ]
```

Comment:
<p>There's nothing _in principle_ that prevents t.my_column.head(1) from working. It's the combination of that expression with almost anything else that doesn't make sense. The technical challenge is that we don't have a straightforward place to put a rewrite rule like</p><p> </p><pre class="ql-syntax" spellcheck="false">t[column].head(N) =&gt; t.head(N)[column]
</pre><p> </p><p>Is there some language we can add here to indicate that this particular operation isn't a fundamental limitation of ibis?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6447/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
This is fine for now!